<a href="https://colab.research.google.com/github/Julio-M39/PesquisaAltosEnem/blob/master/PesquisaAltosEnem.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# **PESQUISA DE DESEMPENHO DOS INSCRITOS ALTOENSE NO ENEM 2016 a 2019 QUE ESTUDARAM EM ESCOLAS PÚBLICAS NO MUNICÍPIO DE ALTOS NO ESTADO DO PIAUÍ**

## **1.0 Bibliotecas**

In [625]:
#Importando Bibliotecas
import pandas as pd

## **2.0 Lendo os Dados**

In [626]:
#Selecionando as bases de dados
fonte2016 = 'https://github.com/Julio-M39/PesquisaAltosEnem/blob/master/microdadosEnem2016.csv?raw=true'
fonte2017 = 'https://github.com/Julio-M39/PesquisaAltosEnem/blob/master/microdadosEnem2017.csv?raw=true'
fonte2018 = 'https://github.com/Julio-M39/PesquisaAltosEnem/blob/master/microdadosEnem2018.csv?raw=true'
fonte2019 = 'https://github.com/Julio-M39/PesquisaAltosEnem/blob/master/microdadosEnem2019.csv?raw=true'

In [627]:
#Leitura de dados
microdadosEnem2016 = pd.read_csv(fonte2016)
microdadosEnem2017 = pd.read_csv(fonte2017)
microdadosEnem2018 = pd.read_csv(fonte2018)
microdadosEnem2019 = pd.read_csv(fonte2019)

## **3.0 Explorando os Dados**

### **3.1 Visualização do Dataframe**

In [628]:
#Visualização do Dataframe
microdadosEnem2019.head(3)

Unnamed: 0  NU_INSCRICAO  NU_ANO  ... NU_NOTA_LC NU_NOTA_MT  NU_NOTA_REDACAO
0       91345  190001095974    2019  ...      562.8      516.0            640.0
1       91355  190001095984    2019  ...      539.0      462.5            560.0
2       91397  190001096026    2019  ...      455.3      413.3            720.0

[3 rows x 17 columns]

Visualização de apenas um Dataframe pois os outros seguem o mesmo padrão de colunas, ou seja as colunas de todos os Dataframes são iguais

### **3.2 Números de inscritos Altoenses por ano**

In [629]:
#Verificando se existem inscrições em branco
print(microdadosEnem2019['NU_INSCRICAO'].isnull().sum())


0


Todas as linhas de inscritos estão preenchidas

In [630]:
#Número de inscritos
print(microdadosEnem2019['NU_INSCRICAO'].shape,
      microdadosEnem2018['NU_INSCRICAO'].shape,
      microdadosEnem2017['NU_INSCRICAO'].shape,
      microdadosEnem2016['NU_INSCRICAO'].shape)

(1507,) (1439,) (1979,) (2450,)


Números de inscritos que residem em Altos por ano:

* Ano de 2019 - 1507 Inscritos
* Ano de 2018 - 1439 Inscritos
* Ano de 2017 - 1979 Inscritos
* Ano de 2016 - 2450 Inscritos

### **3.3 Números de inscritos Altoenses pelo município da escola onde estudaram**

#### **3.3.1 Substituindo os valores da coluna NO_MUNICIPIO_ESC**

No Dataframe original a coluna **NO_MUNICIPIO_ESC** contêm os nomes dos municípios onde os inscritos altoenses estudaram, aqueles que não responderam o campo na coluna fica com valor igual a NaN

In [631]:
#Trocando os campos sem valores (NaN por Não Respondeu)
microdadosEnem2019['NO_MUNICIPIO_ESC'].fillna('Não Respondeu',  inplace=True)
microdadosEnem2018['NO_MUNICIPIO_ESC'].fillna('Não Respondeu',  inplace=True)
microdadosEnem2017['NO_MUNICIPIO_ESC'].fillna('Não Respondeu',  inplace=True)
microdadosEnem2016['NO_MUNICIPIO_ESC'].fillna('Não Respondeu',  inplace=True)

Trocando os campos vazios por "Não Respondeu"

In [632]:
#Novos valores
microdadosEnem2019['NO_MUNICIPIO_ESC'].value_counts()


Não Respondeu          1217
Altos                   233
Teresina                 47
Campo Maior               7
Pau D'Arco do Piauí       2
Bom Jesus                 1
Name: NO_MUNICIPIO_ESC, dtype: int64

#### **3.3.2 Números de inscritos Altoenses pelo município da escola onde estudaram**

##### **2019**

In [633]:
#Verificar os nomes dos municípios onde os inscritos Altoenses estudaram em 2019
microdadosEnem2019['NO_MUNICIPIO_ESC'].value_counts()

Não Respondeu          1217
Altos                   233
Teresina                 47
Campo Maior               7
Pau D'Arco do Piauí       2
Bom Jesus                 1
Name: NO_MUNICIPIO_ESC, dtype: int64

Em 2019 os municípios onde os Altoenses estudaram foram:

* 233 Inscritos - Município de Altos
* 47 Inscritos - Município de Teresina 
* 7 Inscritos - Município de Campo Maior
* 2 Inscritos - Município de Pau D'Arco do Piauí
* 1 Inscritos - Município de Bom Jesus

1217 Inscritos não responderam com o nome do município da escola onde estudou ou estuda, totalizando os 1507 inscritos no ano de 2019.





##### **2018**

In [634]:
#Verificar os nomes dos municípios onde os inscritos Altoenses estudaram em 2018
microdadosEnem2018['NO_MUNICIPIO_ESC'].value_counts()

Não Respondeu          1132
Altos                   257
Teresina                 44
Pau D'Arco do Piauí       3
Campo Maior               2
São João da Serra         1
Name: NO_MUNICIPIO_ESC, dtype: int64

Em 2018 os municípios onde os Altoenses estudaram foram:

* 257 Inscritos - Município de Altos
* 44 Inscritos - Município de Teresina
* 3 Inscritos - Município de Pau D'Arco do Piauí
* 2 Inscritos - Município de Campo Maior
* 1 Inscritos - Município de São João da Serra

1132 Inscritos não responderam com o nome do município da escola onde estudou ou estuda, totalizando os 1439 inscritos no ano de 2018.

##### **2017**

In [635]:
#Verificar os nomes dos municípios onde os inscritos Altoenses estudaram em 2017
microdadosEnem2017['NO_MUNICIPIO_ESC'].value_counts()

Não Respondeu          1564
Altos                   372
Teresina                 34
Pau D'Arco do Piauí       4
Campo Maior               4
Coivaras                  1
Name: NO_MUNICIPIO_ESC, dtype: int64

Em 2017 os municípios onde os Altoenses estudaram foram:

* 372 Inscritos - Município de Altos
* 34 Inscritos - Município de Teresina
* 4 Inscritos - Município de Pau D'Arco do Piauí
* 4 Inscritos - Município de Campo Maior
* 1 Inscritos - Município de Coivaras

1564 Inscritos não responderam com o nome do município da escola onde estudou ou estuda, totalizando os 1979 inscritos no ano de 2017.

##### **2016**

In [636]:
#Verificar os nomes dos municípios onde os inscritos Altoenses estudaram em 2016
microdadosEnem2016['NO_MUNICIPIO_ESC'].value_counts()

Não Respondeu          1983
Altos                   409
Teresina                 42
Pau D'Arco do Piauí      13
Beneditinos               1
Corrente                  1
Santa Inês                1
Name: NO_MUNICIPIO_ESC, dtype: int64

Em 2016 os municípios onde os Altoenses estudaram foram:

* 409 Inscritos - Município de Altos
* 42 Inscritos - Município de Teresina
* 13 Inscritos - Município de Pau D'Arco do Piauí
* 1 Inscrito - Município de Corrente
* 1 Inscritos - Município de Santa Inês
* 1 Inscritos - Município de Beneditinos

1983 Inscritos não responderam com o nome do município da escola onde estudou ou estuda, totalizando os 2450 inscritos no ano de 2017.

### **3.4 Números de inscritos Altoenses pelo tipo de escola**

#### **3.4.1 Substituindo os valores da coluna TP_ESCOLA**

No Dataframe a coluna TP_ESCOLA contêm valores de 1 a 4 para representar os tipos das escolas, sendo eles:

* 1 - Não Respondeu
* 2 - Pública
* 3 - Privada
* 4 - Exterior


In [637]:
#Substituindo os valores
microdadosEnem2019['TP_ESCOLA'] = microdadosEnem2019['TP_ESCOLA'].map({1: 'Não Respondeu' , 2: 'Pública', 3: 'Privada', 4: 'Exterior'})
microdadosEnem2018['TP_ESCOLA'] = microdadosEnem2018['TP_ESCOLA'].map({1: 'Não Respondeu' , 2: 'Pública', 3: 'Privada', 4: 'Exterior'})
microdadosEnem2017['TP_ESCOLA'] = microdadosEnem2017['TP_ESCOLA'].map({1: 'Não Respondeu' , 2: 'Pública', 3: 'Privada', 4: 'Exterior'})
microdadosEnem2016['TP_ESCOLA'] = microdadosEnem2016['TP_ESCOLA'].map({1: 'Não Respondeu' , 2: 'Pública', 3: 'Privada', 4: 'Exterior'})


In [638]:
#Novos valores
microdadosEnem2019['TP_ESCOLA'].value_counts()

Não Respondeu    1160
Pública           325
Privada            22
Name: TP_ESCOLA, dtype: int64

####**3.4.2 Números de inscritos Altoenses pelo tipo de escola por ano:**

##### **2019**

In [639]:
#Números de inscritos pelo tipo de escola
microdadosEnem2019['TP_ESCOLA'].value_counts()

Não Respondeu    1160
Pública           325
Privada            22
Name: TP_ESCOLA, dtype: int64

No ano de 2019 os inscritos Altoenses estudaram em escolas do tipo:

* 325 Inscritos - Colégio Público
* 22 Inscritos - Colégio Privado

1160 inscritos não respondeu em que tipo de escola estudaram totalizando os 1507 Altoenses inscritos no ano de 2019.

##### **2018**

In [640]:
#Números de inscritos pelo tipo de escola
microdadosEnem2018['TP_ESCOLA'].value_counts()

Não Respondeu    1145
Pública           266
Exterior           26
Privada             2
Name: TP_ESCOLA, dtype: int64

No ano de 2018 os inscritos Altoenses estudaram em escolas do tipo:

* 266 Inscritos - Colégio Público
* 26 Inscritos - Colégio Exterior
* 2 Inscritos - Colégio Privado

1145 inscritos não respondeu em que tipo de escola estudaram totalizando os 1439 Altoenses inscritos no ano de 2018.

##### **2017**

In [641]:
#Números de inscritos pelo tipo de escola
microdadosEnem2017['TP_ESCOLA'].value_counts()

Não Respondeu    1564
Pública           383
Privada            32
Name: TP_ESCOLA, dtype: int64

No ano de 2017 os inscritos Altoenses estudaram em escolas do tipo:

* 383 Inscritos - Colégio Público
* 32 Inscritos - Colégio Privado

1564 inscritos não respondeu em que tipo de escola estudaram totalizando os 1979 Altoenses inscritos no ano de 2018.

##### **2016**

In [642]:
#Números de inscritos pelo tipo de escola
microdadosEnem2016['TP_ESCOLA'].value_counts()

Não Respondeu    1983
Pública           424
Privada            43
Name: TP_ESCOLA, dtype: int64

No ano de 2016 os inscritos Altoenses estudaram em escolas do tipo:

* 424 Inscritos - Colégio Público
* 43 Inscritos - Colégio Privado

1983 inscritos não respondeu em que tipo de escola estudaram totalizando os 2450 Altoenses inscritos no ano de 2016.

## **4.0 Limpando dados vazios das notas**

Nas colunas das notas, algumas linhas possuem valores vazios, existem dois motivos para existir esse vazio nos dados:

* 1º Motivo: O candidato faltou a prova
* 2º Motivo: O candidato foi eliminado da prova

Ou seja, esses candidatos não pontuaram

In [643]:
#Visualizar o Dataframe
microdadosEnem2019.head(2)

Unnamed: 0  NU_INSCRICAO  NU_ANO  ... NU_NOTA_LC NU_NOTA_MT  NU_NOTA_REDACAO
0       91345  190001095974    2019  ...      562.8      516.0            640.0
1       91355  190001095984    2019  ...      539.0      462.5            560.0

[2 rows x 17 columns]

In [644]:
#Verificar números de dados vazios
microdadosEnem2019[['NU_NOTA_CN', 'NU_NOTA_CH', 'NU_NOTA_LC', 'NU_NOTA_MT', 'NU_NOTA_REDACAO']].isnull().sum()


NU_NOTA_CN         331
NU_NOTA_CH         279
NU_NOTA_LC         279
NU_NOTA_MT         331
NU_NOTA_REDACAO    279
dtype: int64

Em 2019:

* Nota de Ciências da Natureza tem 331 dados vazios
* Nota de Ciências Humanas tem 279 dados vazios
* Nota de Linguagens e Códigos tem 279 dados vazios
* Nota de Matematica tem 331 dados vazios
* Nota de Redação tem 279 dados vazios

In [645]:
#Verificar números de dados vazios
microdadosEnem2018[['NU_NOTA_CN', 'NU_NOTA_CH', 'NU_NOTA_LC', 'NU_NOTA_MT', 'NU_NOTA_REDACAO']].isnull().sum()


NU_NOTA_CN         298
NU_NOTA_CH         245
NU_NOTA_LC         245
NU_NOTA_MT         298
NU_NOTA_REDACAO    245
dtype: int64

Em 2018:

* Nota de Ciências da Natureza tem 298 dados vazios
* Nota de Ciências Humanas tem 245 dados vazios
* Nota de Linguagens e Códigos tem 245 dados vazios
* Nota de Matematica tem 298 dados vazios
* Nota de Redação tem 245 dados vazios

In [646]:
#Verificar números de dados vazios
microdadosEnem2017[['NU_NOTA_CN', 'NU_NOTA_CH', 'NU_NOTA_LC', 'NU_NOTA_MT', 'NU_NOTA_REDACAO']].isnull().sum()


NU_NOTA_CN         526
NU_NOTA_CH         478
NU_NOTA_LC         478
NU_NOTA_MT         526
NU_NOTA_REDACAO    478
dtype: int64

Em 2017:

* Nota de Ciências da Natureza tem 526 dados vazios
* Nota de Ciências Humanas tem 478 dados vazios
* Nota de Linguagens e Códigos tem 478 dados vazios
* Nota de Matematica tem 526 dados vazios
* Nota de Redação tem 478 dados vazios

In [647]:
#Verificar números de dados vazios
microdadosEnem2016[['NU_NOTA_CN', 'NU_NOTA_CH', 'NU_NOTA_LC', 'NU_NOTA_MT', 'NU_NOTA_REDACAO']].isnull().sum()


NU_NOTA_CN         504
NU_NOTA_CH         504
NU_NOTA_LC         545
NU_NOTA_MT         545
NU_NOTA_REDACAO    545
dtype: int64

Em 2016:

* Nota de Ciências da Natureza tem 504 dados vazios
* Nota de Ciências Humanas tem 504 dados vazios
* Nota de Linguagens e Códigos tem 545 dados vazios
* Nota de Matematica tem 545 dados vazios
* Nota de Redação tem 545 dados vazios

In [648]:
#Criando um novo Dataframe com os valores do Dataframe original
microdadosEnem2019_limpo = microdadosEnem2019
microdadosEnem2018_limpo = microdadosEnem2018
microdadosEnem2017_limpo = microdadosEnem2017
microdadosEnem2016_limpo = microdadosEnem2016


In [649]:
#Removendo os valores vazios do novo Dataframe
microdadosEnem2019_limpo = microdadosEnem2019_limpo.dropna(subset=['NU_NOTA_CN', 'NU_NOTA_CH', 'NU_NOTA_LC','NU_NOTA_MT','NU_NOTA_REDACAO'])
microdadosEnem2018_limpo = microdadosEnem2018_limpo.dropna(subset=['NU_NOTA_CN', 'NU_NOTA_CH', 'NU_NOTA_LC','NU_NOTA_MT','NU_NOTA_REDACAO'])
microdadosEnem2017_limpo = microdadosEnem2017_limpo.dropna(subset=['NU_NOTA_CN', 'NU_NOTA_CH', 'NU_NOTA_LC','NU_NOTA_MT','NU_NOTA_REDACAO'])
microdadosEnem2016_limpo = microdadosEnem2016_limpo.dropna(subset=['NU_NOTA_CN', 'NU_NOTA_CH', 'NU_NOTA_LC','NU_NOTA_MT','NU_NOTA_REDACAO'])


In [650]:
#Visualizando os Dados
microdadosEnem2019_limpo[['NU_NOTA_CN', 'NU_NOTA_CH', 'NU_NOTA_LC','NU_NOTA_MT','NU_NOTA_REDACAO']].isnull().sum()


NU_NOTA_CN         0
NU_NOTA_CH         0
NU_NOTA_LC         0
NU_NOTA_MT         0
NU_NOTA_REDACAO    0
dtype: int64

Todos os valores vazios em 2019 foram retirados

In [651]:
 #Visualizando os Dados
 microdadosEnem2018_limpo[['NU_NOTA_CN', 'NU_NOTA_CH', 'NU_NOTA_LC','NU_NOTA_MT','NU_NOTA_REDACAO']].isnull().sum()

NU_NOTA_CN         0
NU_NOTA_CH         0
NU_NOTA_LC         0
NU_NOTA_MT         0
NU_NOTA_REDACAO    0
dtype: int64

Todos os valores vazios em 2018 foram retirados

In [652]:
#Visualizando os Dados
microdadosEnem2017_limpo[['NU_NOTA_CN', 'NU_NOTA_CH', 'NU_NOTA_LC','NU_NOTA_MT','NU_NOTA_REDACAO']].isnull().sum()

NU_NOTA_CN         0
NU_NOTA_CH         0
NU_NOTA_LC         0
NU_NOTA_MT         0
NU_NOTA_REDACAO    0
dtype: int64

Todos os valores vazios em 2017 foram retirados

In [653]:
#Visualizando os Dados
microdadosEnem2016_limpo[['NU_NOTA_CN', 'NU_NOTA_CH', 'NU_NOTA_LC','NU_NOTA_MT','NU_NOTA_REDACAO']].isnull().sum()

NU_NOTA_CN         0
NU_NOTA_CH         0
NU_NOTA_LC         0
NU_NOTA_MT         0
NU_NOTA_REDACAO    0
dtype: int64

Todos os valores vazios em 2016 foram retirados

In [654]:
#Tamanho do nosso Dataframe original com as notas vazias
print(microdadosEnem2019.shape,
      microdadosEnem2018.shape,
      microdadosEnem2017.shape,
      microdadosEnem2016.shape,)

(1507, 17) (1439, 17) (1979, 17) (2450, 17)


In [655]:
#Tamanho do nosso novo Dataframe sem as notas vazias
print(microdadosEnem2019_limpo.shape,
      microdadosEnem2018_limpo.shape,
      microdadosEnem2017_limpo.shape,
      microdadosEnem2016_limpo.shape,)

(1175, 17) (1139, 17) (1451, 17) (1905, 17)


## **5.0 Novo Dataframe**

Os novos dataframes vão conter apenas os alunos que pontuaram em todas as provas, estudaram em escolas do município de altos e em escolas públicas e privadas

### **5.1 Criando a coluna com as médias dos inscritos**

A coluna MEDIA contêm as (NotaCiênciasNatureza + NotaCiênciasHumanas + NotaLinguagensCódigos + NotaMatematica + NotaRedação)/5

In [678]:
#Criando a nova coluna MEDIA
microdadosEnem2019_limpo['MEDIA'] = (microdadosEnem2019_limpo['NU_NOTA_CN'] + microdadosEnem2019_limpo['NU_NOTA_CH'] + microdadosEnem2019_limpo['NU_NOTA_LC'] + microdadosEnem2019_limpo['NU_NOTA_MT'] + microdadosEnem2019_limpo['NU_NOTA_REDACAO'])/5
microdadosEnem2019_limpo.head(3)

/usr/local/lib/python3.6/dist-packages/ipykernel_launcher.py:2: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  


Unnamed: 0  NU_INSCRICAO  NU_ANO  ... NU_NOTA_MT NU_NOTA_REDACAO   MEDIA
0       91345  190001095974    2019  ...      516.0           640.0  532.24
1       91355  190001095984    2019  ...      462.5           560.0  485.26
2       91397  190001096026    2019  ...      413.3           720.0  491.50

[3 rows x 18 columns]

### **5.2 Visualização do dataframe**

In [657]:
#Visualização
microdadosEnem2019_limpo.head(3)

Unnamed: 0  NU_INSCRICAO  NU_ANO  ... NU_NOTA_MT NU_NOTA_REDACAO   MEDIA
0       91345  190001095974    2019  ...      516.0           640.0  532.24
1       91355  190001095984    2019  ...      462.5           560.0  485.26
2       91397  190001096026    2019  ...      413.3           720.0  491.50

[3 rows x 18 columns]

In [658]:
microdadosEnem2019_limpo['NO_MUNICIPIO_ESC'].value_counts()

Não Respondeu          923
Altos                  198
Teresina                46
Campo Maior              7
Pau D'Arco do Piauí      1
Name: NO_MUNICIPIO_ESC, dtype: int64

### **5.3 Dataframe apenas com inscritos altoenses que responderam ter estudado no município de Altos**

In [659]:
#Criando novos dataframes
DF_microdadosEnem2019 = microdadosEnem2019_limpo.loc[microdadosEnem2019_limpo['NO_MUNICIPIO_ESC'].isin(['Altos'])]
DF_microdadosEnem2018 = microdadosEnem2018_limpo.loc[microdadosEnem2018_limpo['NO_MUNICIPIO_ESC'].isin(['Altos'])]
DF_microdadosEnem2017 = microdadosEnem2017_limpo.loc[microdadosEnem2017_limpo['NO_MUNICIPIO_ESC'].isin(['Altos'])]
DF_microdadosEnem2016 = microdadosEnem2016_limpo.loc[microdadosEnem2016_limpo['NO_MUNICIPIO_ESC'].isin(['Altos'])]

Dataframe novo que contêm apenas os inscritos que responderam que estudaram em colégios no município de Altos

In [679]:
#Quantidade de inscritos
print(DF_microdadosEnem2019.shape,
DF_microdadosEnem2018.shape,
DF_microdadosEnem2017.shape,
DF_microdadosEnem2016.shape)

(198, 18) (220, 17) (311, 17) (360, 17)


Quantidade de inscritos Altoense que estudaram em colégios no município de Altos

* Ano 2019 - 198 inscritos
* Ano 2018 - 220 inscritos
* Ano 2017 - 311 inscritos
* Ano 2016 - 360 inscritos

#### **Novo dataframe 2019**

In [660]:
#Número de inscritos 2019
print(DF_microdadosEnem2019['NO_MUNICIPIO_ESC'].value_counts(),
      DF_microdadosEnem2019['TP_ESCOLA'].value_counts() )

Altos    198
Name: NO_MUNICIPIO_ESC, dtype: int64 Pública    193
Privada      5
Name: TP_ESCOLA, dtype: int64


198 Inscritos altoense estudaram no município de Altos em 2019:

* Públicas - 193 inscritos
* Privadas - 5 inscritos

Totalizando os 198 inscritos

#### **Novo dataframe 2018**

In [661]:
#Número de inscritos 2018
print(DF_microdadosEnem2018['NO_MUNICIPIO_ESC'].value_counts(),
      DF_microdadosEnem2018['TP_ESCOLA'].value_counts() )

Altos    220
Name: NO_MUNICIPIO_ESC, dtype: int64 Pública     208
Exterior     11
Privada       1
Name: TP_ESCOLA, dtype: int64


220 Inscritos altoense estudaram no município de Altos em 2018:

* Públicas - 208 inscritos
* Exterior - 11 inscritos
* Privadas - 1 inscrito

Totalizando os 220 inscritos

#### **Novo dataframe 2017**

In [662]:
#Número de inscritos 2017
print(DF_microdadosEnem2017['NO_MUNICIPIO_ESC'].value_counts(),
      DF_microdadosEnem2017['TP_ESCOLA'].value_counts() )

Altos    311
Name: NO_MUNICIPIO_ESC, dtype: int64 Pública    292
Privada     19
Name: TP_ESCOLA, dtype: int64


311 Inscritos altoense estudaram no município de Altos em 2017:

* Públicas - 292 inscritos
* Privadas - 19 inscritos

Totalizando os 311 inscritos

#### **Novo dataframe 2016**

In [663]:
#Número de inscritos 2016
print(DF_microdadosEnem2016['NO_MUNICIPIO_ESC'].value_counts(),
      DF_microdadosEnem2016['TP_ESCOLA'].value_counts() )

Altos    360
Name: NO_MUNICIPIO_ESC, dtype: int64 Pública    332
Privada     28
Name: TP_ESCOLA, dtype: int64


360 Inscritos altoense estudaram no município de Altos em 2016:

* Públicas - 332 inscritos
* Privadas - 28 inscritos

Totalizando os 360 inscritos

### **5.4 Dataframe apenas com inscritos que estudaram em colégios públicos/privados no município de Altos**

In [666]:
#Criando novos dataframes
DF1_microdadosEnem2019 = DF_microdadosEnem2019.loc[DF_microdadosEnem2019['TP_ESCOLA'].isin(['Pública','Privada'])]
DF1_microdadosEnem2018 = DF_microdadosEnem2018.loc[DF_microdadosEnem2018['TP_ESCOLA'].isin(['Pública','Privada'])]
DF1_microdadosEnem2017 = DF_microdadosEnem2017.loc[DF_microdadosEnem2017['TP_ESCOLA'].isin(['Pública','Privada'])]
DF1_microdadosEnem2016 = DF_microdadosEnem2016.loc[DF_microdadosEnem2016['TP_ESCOLA'].isin(['Pública','Privada'])]

Dataframe que contêm apenas os inscritos que estudaram em colégios públicos/privados no município de Altos

In [680]:
#Quantidade de inscritos
print(DF1_microdadosEnem2019.shape,
DF1_microdadosEnem2018.shape,
DF1_microdadosEnem2017.shape,
DF1_microdadosEnem2016.shape)

(198, 18) (209, 17) (311, 17) (360, 17)


Quantidade de inscritos Altoense que estudaram em colégios no município de Altos em escolas públicas e privadas

* Ano 2019 - 198 inscritos
* Ano 2018 - 209 inscritos
* Ano 2017 - 311 inscritos
* Ano 2016 - 360 inscritos


#### **Novo dataframe 2019**

In [667]:
print(DF1_microdadosEnem2019['NO_MUNICIPIO_ESC'].value_counts(),
      DF1_microdadosEnem2019['TP_ESCOLA'].value_counts())


Altos    198
Name: NO_MUNICIPIO_ESC, dtype: int64 Pública    193
Privada      5
Name: TP_ESCOLA, dtype: int64


198 Inscritos altoense estudaram no município de Altos em 2019:

* Públicas - 193 inscritos
* Privadas - 5 inscritos

Totalizando os 198 inscritos



#### **Novo dataframe 2018**

In [668]:
print(DF1_microdadosEnem2018['NO_MUNICIPIO_ESC'].value_counts(),
      DF1_microdadosEnem2018['TP_ESCOLA'].value_counts())

Altos    209
Name: NO_MUNICIPIO_ESC, dtype: int64 Pública    208
Privada      1
Name: TP_ESCOLA, dtype: int64


209 Inscritos altoense estudaram no município de Altos em 2018:

* Públicas - 208 inscritos
* Privadas - 1 inscrito

Totalizando os 209 inscritos

#### **Novo dataframe 2017**

In [669]:
print(DF1_microdadosEnem2017['NO_MUNICIPIO_ESC'].value_counts(),
      DF1_microdadosEnem2017['TP_ESCOLA'].value_counts())

Altos    311
Name: NO_MUNICIPIO_ESC, dtype: int64 Pública    292
Privada     19
Name: TP_ESCOLA, dtype: int64


311 Inscritos altoense estudaram no município de Altos em 2017:

* Públicas - 292 inscritos
* Privadas - 19 inscritos

Totalizando os 311 inscritos

#### **Novo dataframe 2016**

In [670]:
print(DF1_microdadosEnem2016['NO_MUNICIPIO_ESC'].value_counts(),
      DF1_microdadosEnem2016['TP_ESCOLA'].value_counts())

Altos    360
Name: NO_MUNICIPIO_ESC, dtype: int64 Pública    332
Privada     28
Name: TP_ESCOLA, dtype: int64


360 Inscritos altoense estudaram no município de Altos em 2016:

* Públicas - 332 inscritos
* Privadas - 28 inscritos

Totalizando os 360 inscritos